src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/ProfilerHelper.cs

```diff
@@ -24,8 +24,17 @@ public static class ProfilerHelper
         public const string ClrEnvVarProfiler = ClrEnvVarPrefix + "PROFILER";
         public const string ClrEnvVarProfilerPath = ClrEnvVarPrefix + "PROFILER_PATH";
 
-        public static string GetPath(Architecture architecture) =>
-            NativeLibraryHelper.GetSharedLibraryPath(architecture, ProfilerIdentifiers.LibraryRootFileName);
+        public enum ProfilerVariant
```

Comment:
Let's unnest this. It seems to be used by a lot of the tests and would be slightly easier to read if it was unnested.